### PR TITLE
Have i been pwned v3

### PR DESCRIPTION
**Pull request by teizenman**
## Status
Ready

## Related Issues
https://github.com/demisto/etc/issues/18102

## Description
Re-write from JS to Python

### CODE REVIEW

File: Integrations/Pwned-V2/Pwned-V2.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+
+''' COMMANDS MANAGER / SWITCH PANEL '''
+
+LOG('Command being called is %s' % (demisto.command()))
+
+try:
+    if demisto.command() == 'test-module':
+        test_module()
+    elif demisto.command() == 'pwned-email' or demisto.command() == 'email':
```
Comment (guykeller) [inline Integrations/Pwned-V2/Pwned-V2.py]: each "elif" should contain exactly one command

File: Integrations/Pwned-V2/Pwned-V2.yml [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+    - contextPath: Domain.DBotScore.Type
+      description: The indicator type.
+      type: String
+    - contextPath: Domain.DBotScore.Vendor
+      description: Vendor used to calculate the score.
+      type: String
+    - contextPath: Domain.DBotScore.Score
+      description: The actual score.
+      type: Number
+    - contextPath: Domain(val.Compromised)
```
Comment (guykeller) [inline Integrations/Pwned-V2/Pwned-V2.yml]: Didn't we say we're changing this path? what are the parenthesis?

Comment (guykeller) [reply Integrations/Pwned-V2/Pwned-V2.yml]: Same goes for other paths with () below

**Comment by bakatzir** [comment]
@teizenman @guykeller guys please also make sure you're following the 5.0 context standart: https://github.com/demisto/etc/issues/17949
Timor i assigned you to this integration.

File: Integrations/Pwned-V2/Pwned-V2.yml [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+  required: false
+  type: 8
+- display: Use system proxy
+  name: proxy
+  required: false
+  type: 8
+description: Checks whether emails or domains have been compromised in recent breaches,
+  using the Have I Been Pwned? service.
+display: Pwned-V2
+name: Pwned-V2
```
Comment (ronykoz) [inline Integrations/Pwned-V2/Pwned-V2.yml]: Fix all the name/displayName/id to: Have I Been Pwned? V2

File: Integrations/Pwned-V2/Pwned-V2.yml [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+- defaultvalue: SUSPICIOUS
+  display: 'Domain Severity: The DBot reputation for compromised domains'
+  name: default_dbot_score_domain
+  options:
+  - SUSPICIOUS
+  - MALICIOUS
+  required: false
+  type: 15
+- defaultvalue: 'true'
+  display: Trust any certificate (insecure)
```
Comment (ronykoz) [inline Integrations/Pwned-V2/Pwned-V2.yml]:
```suggestion
  display: Trust any certificate (not secure)
```

File: TestPlaybooks/playbook-PwnedV2_test.yml [hunk 5]
```diff
@@ -0,0 +1,526 @@
+id: Pwned test
+version: 14
```
Comment (ronykoz) [inline TestPlaybooks/playbook-PwnedV2_test.yml]:
```suggestion
version: -1
```

**Comment by kirbles19** [comment]
@ronykoz Done reviewing the .yml.

**Comment by yardensade** [comment]
Why the title says V3? 🤔

**Pull request merged**